### PR TITLE
Add browser benchmark for readers

### PR DESCRIPTION
**Pull request by foxymiles**
**Public-Facing Changes**
None

**Description**
This adds a browser benchmark for the reader classes. To run it issue the `yarn run bench:web` command.

Screen Shot 2022-08-08 at 7 09 26 AM

### CODE REVIEW

**Review by jhurliman — APPROVED** [review]
It would be helpful to see one additional test case of “All Fields”

**Comment by foxymiles** [comment]
> It would be helpful to see one additional test case of “All Fields”

Do we have an example of that in the previous benchmarks?

**Comment by jhurliman** [comment]
> > It would be helpful to see one additional test case of “All Fields”
> 
> Do we have an example of that in the previous benchmarks?

AFAIK, no

**Comment by foxymiles** [comment]
Ok I think a benchmark on reading all fields should be pretty easy to add.

File: bench/benny.ts [hunk 1]
```diff
@@ -91,139 +85,9 @@ async function makeSuite(testCase: Testcase): Promise<void> {
...
-      void (msg as any).arr[99999];
-    },
-    msgDef: `int8[] arr`,
-    msg: {
-      arr: new Int8Array(100000).fill(3),
-    },
-  });
-
-  await makeSuite({
-    name: "float32 array",
```
Comment (defunctzombie) [inline bench/benny.ts:107]: What happened to this testcase in the new version?

File: bench/benny.ts [hunk 2]
```diff
@@ -91,139 +85,9 @@ async function makeSuite(testCase: Testcase): Promise<void> {
 }
 
 async function main() {
-  await makeSuite({
-    name: "int8 array",
```
Comment (defunctzombie) [inline bench/benny.ts:95]: What happened to this test case in the new version?

File: bench/benny.ts [hunk 3]
```diff
@@ -91,139 +85,9 @@ async function makeSuite(testCase: Testcase): Promise<void> {
...
-      void (msg as any).arr[99999];
-    },
-    msgDef: `float32[] arr`,
-    msg: {
-      arr: new Float32Array(100000).fill(3),
-    },
-  });
-
-  await makeSuite({
-    name: "std_msgs/Header",
```
Comment (defunctzombie) [inline bench/benny.ts:119]: What happened to this test case?

**Review by defunctzombie — APPROVED** [review]

File: bench/benny.ts (hunk 2 above)
Comment (foxymiles) [reply bench/benny.ts:95]: Oops those got lost in the shuffle. I've added this back to `testCases.ts`.

**Pull request merged**